modules/engine/src/modules/picking/color-picking.ts

```diff
@@ -154,7 +154,7 @@ export const picking = {
   source,
   vs,
   fs
-} as const satisfies ShaderModule<PickingProps, PickingUniforms>;
+} as const satisfies ShaderModule<PickingProps, PickingUniforms, {}>;
```

Comment:
Is this the empty bindings? It looks a bit stark... I am thinking that maybe we should type things out for orthogonality / clarity?

```ts
export type PickingBindings = {}
```